test/browser/home-page.test.jsx

```diff
@@ -0,0 +1,37 @@
+var should = require('should');
+var sinon = window.sinon;
+var React = require('react/addons');
+var TestUtils = React.addons.TestUtils;
+
+var stubContext = require('./stub-context.jsx');
+var HomePage = require('../../pages/home.jsx');
+
+describe("HomePage", function() {
+  var blogSection;
+
+  beforeEach(function() {
+    blogSection = stubContext.render(HomePage.BlogSection);
+  });
+
+  describe("blogSection", function() {
+    it("should display featured post", function(done) {
+      process.nextTick(function() {
+        blogSection.getDOMNode().querySelector(".featured-post .entry-title").textContent.should.eql("What’s next for Thimble?");
+        done();
+      });
```

Comment:
@toolness in [this comment](https://github.com/mmmavis/teach.webmaker.org/pull/4#discussion_r31220849) you mentioned about using sinon to trigger the feed callback.  Could you give me more pointers?